### PR TITLE
Miscellaneous static and dynamic content updates

### PR DESCRIPTION
**Pull request by marteki**
This fixes errors in the static content and error messaging for the tool. It also adjusts the dynamic showing/hiding for job placement rates and program type to address bugs.
## Additions
- Missing words or periods as needed per the rules of English grammar 😄 
## Removals
- Content that has been removed due to negotiations
## Changes
- Adjusts the criteria for dynamic showing/hiding of job placement rates
- Adjusts how to determine whether a program offers a degree or a certificate
## Testing

Pull down the branch, and load up a sample URL on localhost.
- If more than the allowed amount is inputted for military tuition assistance, the error message should use the word 'awarded' instead of 'borrowed'.
- The content in the Step 1 Federal Loans section about tax burdens and tax advisors should be gone.
- The content at the end of Step 1 should state "Think about how borrowing $X,XXX this year will affect your future finances" instead of "Think about how $X,XXX this year will affect your future finances".
- The sentence in Step 2 that ends with "to borrow money to purchase a car or a home" should end in a period.
- If not given by the school, the job placement rate paragraph should be hidden, and you should not see a NaN% value in Step 2 for it.
- The question between Step 2 and Step 3 should accurately display whether the program [offers a degree](http://localhost:8000/paying-for-college2/understanding-your-financial-aid-offer/offer/?iped=234492&pid=94&oid=740280139f3238cbc9702c7b0d62e5c238a835d0&book=850&gib=6000&gpl=0&hous=20000&insl=0&insi=1&inst=1&mta=4500&othg=2000&othr=500&parl=0&pelg=4000&perl=0&prvf=0&prvi=8&prvl=0&ppl=0&schg=2000&stag=2000&subl=1000&totl=88592&tran=5000&tuit=30000&unsl=1000&wkst=0) or [offers a certificate](http://localhost:8000/paying-for-college2/understanding-your-financial-aid-offer/offer/?iped=117113&pid=2765&totl=50000&tuit=30000&hous=20000&book=10000&tran=5000&othr=500&pelg=0&schg=2000&stag=3000&othg=2000&mta=2000&gib=5000&wkst=100&ppl=2110&parl=1234&perl=0&subl=0&unsl=0&gpl=0&prvl=0&prvi=20&prvf=10&insl=10000&insi=4&inst=50&oid=9e0280139f3238cbc9702c7b0d62e5c238a835d0).

All JS unit tests should pass.
After some package updates and patching, our NPM security test should pass.
**Functional browser tests are still not passing. They will be re-evaluated and refactored in a coming PR.**
## Review
- @mistergone @higs4281 @amymok @anselmbradford
## Checklist
- [x] Changes are limited to a single goal (no scope creep)
- [x] Code can be automatically merged (no conflicts)
- [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
- [ ] Passes all existing automated tests
- [x] New functions include new tests
- [x] New functions are documented (with a description, list of inputs, and expected output)
- [x] Placeholder code is flagged
- [x] Visually tested in supported browsers and devices
- [x] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: src/disclosures/js/views/financial-view.js [hunk 1]
```diff
@@ -241,11 +241,14 @@ var financialView = {
       $( '.content_graduate-program' ).hide();
       financialView.gradPlusVisible( false );
     }
-    this.jobPlacementVisible( values.jobRate !== '' );
-    if ( values.level.indexOf( 'Certificate' ) === 1 ) {
-      this.$degreeType.text( 'certificate' );
-    } else {
+    this.jobPlacementVisible(
+      values.jobRate !== 'None' && values.jobRate !== '' &&
+       typeof values.jobRate !== 'undefined'
```
Comment (anselmbradford) [inline src/disclosures/js/views/financial-view.js]: Trivial: remove extra space to align `values` and `typeof`. Also you might move the type check to be the first check that happens.

**Comment by marteki** [comment]
> The content at the end of Step 1 should state "Think about how borrowing $X,XXX this year will affect your future finances" instead of "Think about how $X,XXX this year will affect your future finances".

For some reason, the dynamic paragraphs for positive/negative cost remaining aren't showing up for me anymore. 😞  I'm filing a bug on our tracker to look into this more.

File: src/disclosures/js/views/financial-view.js (hunk 1 above)
Comment (marteki) [inline src/disclosures/js/views/financial-view.js]: Cool. The back-end is supposed to gather up all the undefined values and send them to the API with a 'None' value. But yeah, if something fails on that, the `typeof` check should happen first. Thanks for the catch!

**Comment by mistergone** [comment]
LGTM, aside from the bug you mentioned above.

drwho-thumbs-up

**Comment by higs4281** [comment]
- npm tests pass with 4 vulns
- all content test points pass, but I see NaN for this PR where build shows a value.

on [build](http://build.consumerfinance.gov/paying-for-college2/understanding-your-financial-aid-offer/offer/?iped=210960&pid=44&oid=fa8283b5b7c939a058889f997949efa566c616a3&book=1500&gib=0&gpl=0&hous=5611&insi=5.0&insl=3000&inst=36&mta=0&othg=100&othr=6079&parl=7000&pelg=1500&perl=5500&ppl=0&prvl=2000&prvf=2.1&prvi=4.55&schg=2000&stag=2000&subl=3500&totl=91004&tran=500&tuit=16360&unsl=2000&wkst=1000):  

debt_on_build

this PR in standalone:  

debt_pr226

**Comment by mistergone** [comment]
@higs4281 I'm not seeing the same NaN. There's no migrate or anything to do, is there?

**Comment by marteki** [comment]
@higs4281 I also can't replicate that with that program. I'm not even seeing the same number with the default aid offer on build.

Are you inputting any modifications to the given aid offer? If so, what are they, so we can see what might be triggering the NaN for you?

**Comment by higs4281** [comment]
Just copied the string from build:

```
http://localhost:8000/paying-for-college2/understanding-your-financial-aid-offer/offer/?iped=210960&pid=44&oid=fa8283b5b7c939a058889f997949efa566c616a3&book=1500&gib=0&gpl=0&hous=5611&insi=5.0&insl=3000&inst=36&mta=0&othg=100&othr=6079&parl=7000&pelg=1500&perl=5500&ppl=0&prvl=2000&prvf=2.1&prvi=4.55&schg=2000&stag=2000&subl=3500&totl=91004&tran=500&tuit=16360&unsl=2000&wkst=1000
```

**Comment by marteki** [comment]
Same. After a `manage.py migrate`, `manage.py loaddata collegedata`, and `gulp`, this is what I see on localhost with no aid offer inputs or adjustments:

screen shot 2016-07-25 at 12 43 04 pm

and

screen shot 2016-07-25 at 12 43 24 pm

Are you seeing at least the first screenshot? Is our data the same?

**Comment by higs4281** [comment]
Top is same. must be something local for me.

**Comment by higs4281** [comment]
rebuilt branch and got same result. something's odd with that program. here's what I see for Step 1 big picture on [build](http://build.consumerfinance.gov/paying-for-college2/understanding-your-financial-aid-offer/offer/?iped=210960&pid=44&oid=fa8283b5b7c939a058889f997949efa566c616a3&book=1500&gib=0&gpl=0&hous=5611&insi=5.0&insl=3000&inst=36&mta=0&othg=100&othr=6079&parl=7000&pelg=1500&perl=5500&ppl=0&prvl=2000&prvf=2.1&prvi=4.55&schg=2000&stag=2000&subl=3500&totl=91004&tran=500&tuit=16360&unsl=2000&wkst=1000).

build_big_picture

**Comment by marteki** [comment]
Pull this branch again. I merged in the adjustments you just merged for program length. Otherwise, no program length is set in the dropdown, and that has ripple effects down the page for calculations.

**Comment by marteki** [comment]
And build won't have that working until the Jenkins job is rebuilt.

**Comment by higs4281** [comment]
I say ignore my issues. The program data for that example is wildly different from the offer data. Let me get those more in line in the sample URL, but don't let this hold up the PR.
👍

**Comment by marteki** [comment]
Merging!

**Pull request merged**